ansible-playbooks/ros_disable_and_clean_up.yml

```diff
@@ -0,0 +1,27 @@
+---
+# This playbook will take care of all steps required to remove
+# Insights resource-optimization (pcp packages)
+- hosts: all
+  tasks:
+    - name: Ensure a pcp service is stopped
+      systemd:
+        state: stopped
+        enabled: no
+        name: "{{ item }}"
+      with_items:
+        - 'pmcd'
+        - 'pmlogger'
+    - name: Remove ros pmlogger service instance
+      lineinfile:
+        dest: /etc/pcp/pmlogger/control.d/local
+        regexp: "LOCALHOSTNAME	n   y	PCP_LOG_DIR/pmlogger/ros	-r -T24h10m -c config.ros -v 100Mb"
```

Comment:
@patilsuraj767, 
Can't we just check `PCP_LOG_DIR/pmlogger/ros` inside regexp? 
By any chance if user adds/removes extra white spaces might fail to detect. What do you think?